packages/grafana-ui/src/components/Forms/RadioButtonGroup/RadioButtonGroup.tsx

```diff
@@ -85,5 +86,10 @@ const getStyles = (theme: GrafanaTheme2) => {
     icon: css`
       margin-right: 6px;
     `,
+    img: css`
+      width: 16px;
+      height: 16px;
```

Comment:
nit: maybe use spacing
```suggestion
      width: ${theme.spacing(2)};
      height: ${theme.spacing(2)};
```